bootstrap.js

```diff
@@ -97,12 +99,16 @@ function startup(data, reason) {
   Cu.import("resource://tabcenter/verticaltabs.jsm");
   unload(vtInit());
   watchWindows(function(window) {
-    let vt = new VerticalTabs(window);
+    let vt = new VerticalTabs(window, {newPayload, addPingStats});
     unload(vt.unload.bind(vt), window);
   }, "navigator:browser");
+  timer.initWithCallback({notify: () => {
+    sendPing();
+  }}, 24*60*60*1000, Ci.nsITimer.TYPE_REPEATING_SLACK);  // Every 24h…
```

Comment:
What's the reason for this ping? Is this a manual "daily active user" thing? I think we can rely on the test pilot add-on to keep track of installation numbers for us (otherwise I need to open a new PR against the search add-on repo...)